app/controllers/bgeigie_imports/waiting_for_controller.rb

```diff
@@ -0,0 +1,11 @@
+module BgeigieImports
+  class WaitingForController < ApplicationController
+    before_filter :authenticate_user!
+    before_filter :require_moderator
+
+    def index
+      @bgeigie_imports = BgeigieImport.where(status: :waiting_for).page(params[:page])
```

Comment:
`awaiting_response` might be a better word choice than `waiting_for`

Reply:
oh... thank you. renamed including controller name and such